### ISSUE TITLE
Investigate globby options

### ISSUE THREAD

**Issue by hzoo** [babel/babel-upgrade#32]
Ref https://github.com/babel/babel-upgrade/pull/31

https://github.com/sindresorhus/globby#gitignore

I think maybe we can just pass this instead of the explicit node_modules check?

**Comment by Raigen** [comment]
I tried it out with different projects, also with https://github.com/Raigen/babel6-tmp.
As long as babel-upgrade does not support monorepos this could be counterproductive for these. The current solution I did for our monorepos is to manually `cd` into each package inside the monorepo and run babel-upgrade. This updates the `.babelrc` just fine.
When I replace the explicit node_modules check with the gitignore option it only works from the root of the project. From inside the packages it will again find all package.json files in node_modules and do not update the `.babelrc`. It seems globby does not lookup the gitignore in parent folders.

I already have a branch with simple monorepo support. I will try out the gitignore option with that branch tomorrow.

**Comment by hzoo** [comment]
Yeah we don't have to switch was just wondering if it works, looking forward to the monorepo support +1, you could test it on the babel repo too actually
